app/src/components/Pages/Sos/IdentityTable.tsx

```diff
@@ -0,0 +1,56 @@
+
+import { TableDisplayMode } from './TableDisplayMode';
+import axios, { AxiosRequestConfig } from 'axios';
+import { User } from '../../api/configuration/models/users';
+import React, { useState, useEffect } from 'react';
+
+export const serverAPIPort = 3000;
+export const host = 'http://localhost';
+export const APIRootPath = `${host}:${serverAPIPort}`;
+
+const IdentityTable = () => {
+  const [users, setUsers] = useState<User[] | undefined>(undefined);
+
+  const fetchUsers = async () => {
+    const params: AxiosRequestConfig = {
+      baseURL: `${APIRootPath}`,
```

Comment:
Please use process.env.REACT_BACKEND... instead APIEootPath. take a look in http-hook. It necessary because of the deployment. The url is on dev localhost, but in the cloud it's a different url